### DISCUSSION TITLE
what's version number of release tag 1.1.9

### DISCUSSION THREAD

**Discussion by Yuqiulei123** [rustdesk/rustdesk#4456] [General]
I download code from https://github.com/rustdesk/rustdesk/commits/1.1.9, while the program i compiled out is not equal to the version of rustdesk-1.1.9-putes.exe.
I want to know the exactly version number of release tag 1.1.9,thank you.

**Comment by SergeyMy** [comment]
1. why? 1.1.9 is very outdated

2. version 1.2.0 of the master branch builds fine with both **Sciter** and **Flutter**

PS
**tag 1.1.9**
d716e2b4 ("Merge pull request #474 from ripp2003/patch-1", 2022-05-02)
...
build rustdesk-1.1.9-fedora28-centos8.rpm 
Mon **Jun 13 16:41:18 2022**
c9315f3b ("Merge pull request #757 from sitiom/patch-1", 2022-06-13)
... 1.1.10
3f2aaae1 ("opt: merge addon", 2022-06-27)
0d9218ea ("Merge pull request #874 from Kingtous/flutter_desktop", 2022-06-27)

PSPS 1.1.9 bad idea, example:
https://github.com/rustdesk/rustdesk/discussions/4428

**Comment by Yuqiulei123** [comment]
That works,thanks.
